docker/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  snp_blending:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - TAG=humble-master
+    environment:
+      DISPLAY: $DISPLAY
+      XAUTHORITY: $XAUTHORITY
+      NVIDIA_DRIVER_CAPABILITIES: all
+      ROS_LOG_DIR: /tmp
+    container_name: snp_blending
+    image: ghcr.io/ros-industrial-consortium/snp_blending:humble-master
+    stdin_open: true
+    tty: true
+    network_mode: host
+    privileged: true
+    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
```

Add the user to the `sudo` group inside the container

```suggestion
    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
    group_add:
      - sudo
```